examples/todo-typescript/src/server/actions.ts

```diff
@@ -0,0 +1,33 @@
+import HttpError from '@wasp/core/HttpError.js';
+import { Context, Task } from './serverTypes'
+
+type CreateArgs = { description: Task['description'] };
```

Comment:
Oh, something I forgot to tell you. Check [this](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcAqBDAzgazgXjgG8AoOOASwBMAuOAOwFcBbAI2CjLiuCwGMoFMDAoR6dLDEH0A5lwpYAImOB1WECABtgGel0ZYOASVoMW7TgF8SJJCjgBhKDpjAAglBlYCcAAoU+HAAeTFwAGjgAch5+QWFRekiAPiA) out.

TypeScript's utility types are generally very useful and powerful, I recommend learning all of them: https://www.typescriptlang.org/docs/handbook/utility-types.html.

Reply:
oooh nice 
🚀